Selenium_Task80/TestProject2/LoginTests.cs

```diff
@@ -0,0 +1,53 @@
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using Selenium_Task80;
+using System.Threading;
+using System;
+
+namespace LoginTests
+{
+    public class Tests
+    {
+        public IWebDriver driver;
+
+        [SetUp]
+        public void BeginTest()
+        {
+            driver = new ChromeDriver();
+            driver.Url = "https://www.yandex.com";
+            Screenshot screenshot = ((ITakesScreenshot)driver).GetScreenshot();
+            screenshot.SaveAsFile(@"C:\Users\nastyascherbinskaya\Desktop\Screenshot.jpg");
```

Comment:
try to use not hardcoded path. It is better to store screenshots in solution folders like "Screenshots"